lib/WebService/Async/CustomerIO.pm

```diff
@@ -49,14 +50,16 @@ Parameters:
 
 =item * C<api_key>
 
+=item * C<api_token>
+
 =back
 
 =cut
 
 sub _init {
     my ($self, $args) = @_;
 
-    for my $k (qw(site_id api_key)) {
+    for my $k (qw(site_id api_key api_token)) {
```

Comment:
A Customer.IO API Token is required to consume Regular API endpoint with Bearer authentication.